tests/test_predicates.py

```
@@ -190,6 +190,92 @@ def test_memory_in_use_quiet_on_empty_query() -> None:
+    ) is True
+
+
+def test_preference_statement_quiet_on_non_preference_query() -> None:
+    """Probes for false-positive triggers — common scientific phrasings
+    that contain ``以后``/``默认``/``always`` as time/state adverbs rather
+    than expressed preferences. These must NOT fire."""
+    from omicsclaw.runtime.predicates import preference_statement_intent
+
+    assert preference_statement_intent(_req(query="run sc-de")) is False
+    assert preference_statement_intent(_req(query="explain UMAP")) is False
+    # "以后" + 时间名词（不是动词）
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)